docs/tags.rst

```diff
@@ -92,11 +94,108 @@ Reference
 
     The function returns an iterable in order to allow for the possible
     short-circuiting of tag generation if the entire sequence is not necessary
-    and calculating some tags happens to be expensive.
+    and tag calculation happens to be expensive.
 
     :param bool warn: Whether warnings should be logged. Defaults to ``False``.
 
 
+.. function:: interpreter_name()
+
+    Returns the running interpreter's name.
+
+    This typically acts as the prefix to the :attr:`~Tag.interpreter` tag.
+
+
+.. function:: interpreter_version(*, warn=False)
+
+    Returns the running interpreter's version.
+
+    This typically acts as the suffix to the :attr:`~Tag.interpreter` tag.
+
+
+.. function:: mac_platforms(version=None, arch=None)
+
+    Yields the :attr:`~Tag.platform` tags for macOS.
+
+    :param tuple version: A two-item tuple presenting the version of macOS.
+                          Defaults to the current system's version.
+    :param str arch: The CPU architecture. Defaults to the architecture of the
+                     current system, e.g. ``"x86_64"``.
+
+    .. note::
+        Equivalent support for the other major platforms is purposefully not
+        provided:
+
+        - On Windows, platform compatibility is statically specified
+        - On Linux, code must be run on the system itself to determine
+          compatibility
+
+
+.. function:: compatible_tags(python_version=None, interpreter=None, platforms=None)
+
+    Yields the tags for an interpreter compatible with the Python version
+    specified by ``python_version``.
+
+    The specific tags generated are:
+
+    - ``py*-none-<platform>``
+    - ``<interpreter>-none-any`` if ``interpreter`` is provided
+    - ``py*-none-any``
+
+    :param Sequence python_version: A one- or two-item sequence representing the
+                                 compatible version of Python. Defaults to
+                                 ``sys.version_info[:2]``.
+    :param str interpreter: The name of the interpreter (if known), e.g.
+                            ``"cp38"``. Defaults to the current interpreter.
+    :param Iterable platforms: Iterable of compatible platforms. Defaults to the
+                               platforms compatible with the current system.
+
+.. function:: cpython_tags(python_version=None, abis=None, platforms=None, *, warn=False)
+
+    Yields the tags for the CPython interpreter.
+
+    The specific tags generated are:
+
+    - ``cp<python_version>-<abi>-<platform>``
+    - ``cp<python_version>-abi3-<platform>``
+    - ``cp<python_version>-none-<platform>``
+    - ``cp<older version>-abi3-<platform>`` where "older version" is all older
+      minor versions down to Python 3.2 (when ``abi3`` was introduced)
+
```

Comment:
Sorry for the necro, but why is `cp<older version>` only available for `abi3` and not `none`?

Reply:
Please file a new issue for discussing this.

Reply:
I found GH-311, I think I'll continue the discussion there.